src/core/quicdef.h

```diff
@@ -538,6 +538,54 @@ CXPLAT_STATIC_ASSERT(
 //
 #define QUIC_TP_MAX_STREAMS_MAX                             ((1ULL << 60) - 1)
 
+/*************************************************************
```

Comment:
Please remove this duplicate/unnecessary change.

Reply:
PR depends on _global branch made duplicate change by accident...